sw/device/silicon_creator/mask_rom/mask_rom.c

```diff
@@ -177,10 +173,12 @@ void mask_rom_boot(void) {
     }
   }
 
-  // Boot failed for all ROM_EXTs allowed by boot policy
-  // boot_failed(boot_policy); // Boot Policy Module
-  asm volatile("unimp");
+  return kErrorMaskRomBootFailed;
+}
+
+void mask_rom_main(void) {
```

Comment:
This is `noreturn` right?

Reply:
Its prototype in `mask_rom.h` is `noreturn void mask_rom_main(void)`. I'm happy to duplicate the noreturn annotation here if that is best practice.

Reply:
*shrug*.